test/k8s-integration/config/sc-standard.yaml

```diff
@@ -5,4 +5,5 @@ metadata:
 provisioner: pd.csi.storage.gke.io
 parameters:
   type: pd-standard
+  labels: key1=value1,key2=value2
```

Comment:
Maybe add a comment here saying this is for tests?

Reply:
Good idea, done.